.github/workflows/label-actions.yml

```diff
@@ -18,4 +18,4 @@ jobs:
   action:
     runs-on: ubuntu-latest
     steps:
-      - uses: dessant/label-actions@v3
+      - uses: okdas/label-actions@patch-1
```

Comment:
Currently testing my fork on a separate repo so we can then use variables in messages.

Reply:
Opened a PR to add this functionality, but for now let's use my fork.
https://github.com/dessant/label-actions/pull/29